key-share/src/lib.rs

```diff
@@ -119,45 +133,90 @@ pub struct VssSetup<E: Curve> {
     pub I: Vec<NonZero<Scalar<E>>>,
 }
 
+/// Public Key Info
+///
+/// Contains public information about the TSS key, including shared public key, commitments to
+/// secret shares and etc.
+#[derive(Clone, Debug)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "serde", serde(bound = ""))]
+pub struct DirtyKeyInfo<E: Curve> {
```

Comment:
Do we use udigest in this crate yet? Would be convenient to have

Reply:
Yep good idea! Made it digestable